securedrop/config_tool.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import re
+import sys
+import yaml
+
+from os import path
+from yaml.scanner import ScannerError
+
+
+"""
+A CLI tool to help admins configure their SecureDrop instances. Uses argparse, so run this file from
+your terminal with --help to get instructions on how to use this.
+"""
+
+
+def init_logger():
+    l = logging.getLogger(__file__)
+    l.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler(sys.stdout)
+    ch.setLevel(logging.DEBUG)
+    ch.setFormatter(logging.Formatter('%(levelname)s - %(message)s'))
+    l.addHandler(ch)
+    return l
+
+log = init_logger()
+
+
+class ConfigToolException(Exception):
+
+    def __init__(self, msg=None):
+        super(ConfigToolException, self).__init__(msg)
+
+
+def check_ip_address_valid(ip):  # "good enough"
```

Comment:
If we `import ipaddress` (which requires a dependency on py2-ipaddress for python2) then we can use `ipaddress.ip_address(ip)` and check for a `ValueError`, and that would give us free ipv6 support for this check too.